precode.sh

```diff
@@ -6,38 +6,67 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir task
```

Comment:
Можно одной командой
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```